packages/core/scripts/local/DeployEMP.js

```diff
@@ -19,34 +19,29 @@
  * - `./PushPriceEMP.js`: "resolves" a pending mock oracle price request with a price.
  *
  *
- * Example: $(npm bin)/truffle exec ./scripts/local/DeployEMP.js --network test --test true --identifier ETH/BTC
+ * Example: $(npm bin)/truffle exec ./scripts/local/DeployEMP.js --network test --test true --identifier ETH/BTC --cversion 1.1.0
  */
 const { toWei, utf8ToHex, hexToUtf8 } = web3.utils;
 const { interfaceName } = require("@uma/common");
+const { getAbi, getTruffleContract } = require("../../index");
+const argv = require("minimist")(process.argv.slice(), {
+  boolean: ["test"],
+  string: ["identifier", "collateral", "cversion"]
+});
+const abiVersion = argv.cversion || "1.1.0"; // Default to most recent mainnet deployment, 1.1.0.
 
 // Deployed contract ABI's and addresses we need to fetch.
-const ExpiringMultiPartyCreator = artifacts.require("ExpiringMultiPartyCreator");
-const ExpiringMultiParty = artifacts.require("ExpiringMultiParty");
-const Finder = artifacts.require("Finder");
-const IdentifierWhitelist = artifacts.require("IdentifierWhitelist");
-const MockOracle = artifacts.require("MockOracle");
-const TestnetERC20 = artifacts.require("TestnetERC20");
-const WETH9 = artifacts.require("WETH9");
-const Timer = artifacts.require("Timer");
-const TokenFactory = artifacts.require("TokenFactory");
-const AddressWhitelist = artifacts.require("AddressWhitelist");
-const Store = artifacts.require("Store");
-const argv = require("minimist")(process.argv.slice(), { boolean: ["test"], string: ["identifier", "collateral"] });
-const { getAbi } = require("../../index");
-
-// Contracts we need to interact with.
-let collateralToken;
-let emp;
-let mockOracle;
-let identifierWhitelist;
-let collateralTokenWhitelist;
-let expiringMultiPartyCreator;
-let store;
```

Comment:
Note: just for cleanliness, I got rid of these globals since they aren't really necessary since they are _only_ used in a single scope.